packages/services/api/src/modules/target/module.graphql.ts

```diff
@@ -139,43 +172,71 @@ export default gql`
     bySelector: TargetSelectorInput @tag(name: "public")
   }
 
-  input UpdateTargetValidationSettingsInput {
-    organizationSlug: String!
-    projectSlug: String!
-    targetSlug: String!
-    period: Int!
-    percentage: Float!
-    requestCount: Int! = 1
-    breakingChangeFormula: BreakingChangeFormula! = PERCENTAGE
-    targetIds: [ID!]!
-    excludedClients: [String!]
+  """
+  Fields not provided (omitted) will retain the previous value.
+  """
+  input ConditionalBreakingChangeConfigurationInput {
+    """
+    Update whethe the conditional breaking change detection is enabled or disabled.
+    """
+    isEnabled: Boolean @tag(name: "public")
+    """
+    The period in days. Operations of the last x days will be used for the conditional breaking change detection.
+    The maximum value depends on the organizations data retention limits.
+    """
+    period: Int @tag(name: "public")
+    """
+    Whether a percentage or absolute value should be used for the conditional breaking changes treshold.
+    """
+    breakingChangeFormula: BreakingChangeFormulaType @tag(name: "public")
+    """
+    The percentage value if \`UpdateTargetValidationSettingsInput.breakingChangeFormula\` is set to \`BreakingChangeFormulaType.PERCENTAGE\`.
+    """
+    percentage: Float @tag(name: "public")
+    """
+    The request count value if \`UpdateTargetValidationSettingsInput.breakingChangeFormula\` is set to \`BreakingChangeFormulaType.REQUEST_COUNT\`.
+    """
+    requestCount: Int @tag(name: "public")
+    """
+    List of target ids within the same project, whose operations are used for the breaking change detection.
+    """
+    targetIds: [ID!] @tag(name: "public")
+    """
+    List of client names that are excluded from the breaking change detection.
+    """
+    excludedClients: [String!] @tag(name: "public")
+  }
```

Comment:
I did not really see any benefit of having the `isEnabled` and other configuration options separated. Thus I merged the two mutation fields into a single one that can handle both cases.